src/videojs-contrib-hls.js

```diff
@@ -278,8 +279,42 @@ export default class HlsHandler extends Component {
       let seekable;
 
       if (!updatedPlaylist) {
+        let selectedPlaylist = this.selectPlaylist();
+
+        if (!selectedPlaylist.attributes.AUDIO) {
+          // from safari defaults
+          this.tech_.audioTracks().addTrack(new AudioTrack({
+            enabled: true,
+            id: '1',
+            kind: 'main',
+            tech: this.tech_
+          }));
+        } else {
+          let audioMediaGroupName = selectedPlaylist.attributes.AUDIO;
+          let mg = this.playlists.master.mediaGroups.AUDIO[audioMediaGroupName];
+
+          for (let label in mg) {
+            let language = mg[label].language || '';
+            /* eslint-disable dot-notation */
+            // we need to use non dot notation around defaul for ie8
```

Comment:
This project doesn't support IE8 so as long as this doesn't fail at parse-time, it's ok to skip IE8 compatibility stuff.